integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomainNoLB.java

```diff
@@ -0,0 +1,870 @@
+import static oracle.weblogic.kubernetes.utils.PodUtils.setPodAntiAffinity;
+import static oracle.weblogic.kubernetes.utils.SecretUtils.createSecretWithUsernamePassword;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assumptions.assumeFalse;
+
+/**
+ * Verify scaling up and down the clusters in the domain with different domain types.
+ * Also verify the rolling restart behavior in a MII domain.
+ */
+@DisplayName("Verify scaling the clusters in the domain with different domain types, and "
+        + "rolling restart behavior in a MII domain")
+@IntegrationTest
+class ItParameterizedDomainNoLB {
+
+  // domain constants
+  private static final String clusterName = "cluster-1";
+  private static final int replicaCount = 2;
+  private static final String wlSecretName = "weblogic-credentials";
+  private static final String miiDomainUid = "miidomain";
+
+  private static String opNamespace = null;
+  private static List<Domain> domains = new ArrayList<>();
+  private static LoggingFacade logger = null;
+  private static String miiDomainNamespace = null;
+  private static Map<String, Quantity> resourceRequest = new HashMap<>();
+  private static Map<String, Quantity> resourceLimit = new HashMap<>();
+
+  /**
+   * Install operator.
+   * Create three different type of domains: model in image, domain on PV and domain in image.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(4) List<String> namespaces) {
+    logger = getLogger();
+
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get unique namespaces for three different type of domains
+    logger.info("Getting unique namespaces for three different type of domains");
+    assertNotNull(namespaces.get(1));
+    miiDomainNamespace = namespaces.get(1);
+    assertNotNull(namespaces.get(2));
+    String domainOnPVNamespace = namespaces.get(2);
+    assertNotNull(namespaces.get(3));
+    String domainInImageNamespace = namespaces.get(3);
+
+    // set the service account name for the operator
+    String opServiceAccount = opNamespace + "-sa";
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, opServiceAccount, false, 0,
+        miiDomainNamespace, domainOnPVNamespace, domainInImageNamespace);
+
+    // set resource request and limit
```

Comment:
Cann't we use the default. We may be running into resource issue by configuring high cpu/memory limit.

Reply:
fixed